tests/test_table.py

```diff
@@ -74,10 +81,17 @@ def test_set(self):
         with self.assertRaises(WasmtimeError):
             table.set(store, 1, func)
 
-    def test_errors(self):
-        ty = TableType(ValType.i32(), Limits(1, 2))
-        with self.assertRaises(AttributeError):
-            Table(1, ty, 2)  # type: ignore
-        store = Store()
-        with self.assertRaises(AttributeError):
-            Table(store, 2, 2)  # type: ignore
+    """
+    The following test case currently causes the following runtime error:
+
+    Fthread '<unnamed>' panicked at 'Attempt to convert non-reference type to a reference type', crates/wasmtime/src/types.rs:315:26
+
+    TODO: Figure out if this should be enabled and how to properly handle this.
+    """
+    # def test_errors(self):
+    #     ty = TableType(ValType.i32(), Limits(1, 2))
+    #     with self.assertRaises(AttributeError):
+    #         Table(1, ty, 2)  # type: ignore
+    #     store = Store()
+    #     with self.assertRaises(AttributeError):
+    #         Table(store, 2, 2)  # type: ignore
```

Comment:
Oh both instances of the commented out tests in this file are ok to remove, that was a behavior change in the C API which is expected

Reply:
Ah great, I'll remove them :+1: